Doc/library/datetime.rst

```diff
@@ -1213,7 +1213,7 @@ Supported operations:
 
    If both are naive, or both are aware and have the same :attr:`~.datetime.tzinfo` attribute,
    the :attr:`~.datetime.tzinfo` attributes are ignored, and the result is a :class:`timedelta`
-   object *t* such that ``datetime2 + t == datetime1``. No time zone adjustments
+   object ``t`` such that ``datetime2 + t == datetime1``. No time zone adjustments
```

Comment:
@erlend-aasland I've found several occurrences like these, they don't seem to be method or function *parameters* like you said in https://github.com/python/cpython/issues/85453#issuecomment-2076652463, but I'm not sure if they should all be marked up as variables.

Reply:
I think these fall in the variables/literals/code category of our style guide:

https://devguide.python.org/documentation/markup/#quick-reference